plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-device.c

```diff
@@ -275,11 +274,10 @@ fu_logitech_bulkcontroller_device_send_sync_cmd(FuLogitechBulkcontrollerDevice *
 						GByteArray *buf,
 						GError **error)
 {
-	gchar *verify_buf;
 	guint32 cmd_tmp = 0x0;
 	guint64 cmd_tmp_64 = 0x0;
-	gchar *pEnd;
-	guint32 verify_cmd;
+	guint32 verify_cmd = 0x0;
+	guint8 ack_payload[SYNC_ACK_PAYLOAD_LENGTH];
```

Comment:
can this be `gchar`? also, it might be wise to do `= { 0x0 }` to NUL terminate the string always.

Reply:
Agreed. Fixed